app/views/wpcc/your_contributions/new.html.erb

```diff
@@ -2,11 +2,31 @@
 
 <section class="contributions">
   <h2 class="wpcc__heading">2. <%= t('wpcc.contributions.title') %></h2>
-  <div class="contributions__row">
+  <div class="contributions__row" data-dough-component="PopupTip">
     <p class="t-contributions_description">
-      <%= @your_contribution.earnings_description %>
+      <% if @message_presenter.minimum_contribution? %>
+        <% tooltip_html = render('wpcc/tooltips/trigger', locals: { popup_type: 't-qualifying-earnings' }) %>
+        <%= @your_contribution.earnings_description({ tooltip_html: raw(tooltip_html) }) %>
+      <% else %>
+        <%= @your_contribution.earnings_description %>
+      <% end %>
     </p>
+
+    <div data-dough-popup-container class="popup-tip__container contributions__helper">
+      <p data-dough-popup-content class="popup-tip__content">
+        <span class="popup-tip__title--no-js">
+          <%= t('wpcc.tooltip_qualifying_earnings.title') %>:
+        </span>
+        <%= t(
+          'wpcc.tooltip_qualifying_earnings.text',
+          formatted_upper_earnings: @message_presenter.formatted_upper_earnings,
+          formatted_lower_earnings: @message_presenter.formatted_lower_earnings
+          ) %>
+      </p>
+      <%= render 'wpcc/tooltips/close' %>
+    </div>
```

Comment:
The markup in lines 15-27 is repeated in several places. 

Could it be moved to a partial and use local variables to pass the class name e.g. 'contributions__helper'? 

Same for lines 19-32 in your_results/index.html.erb - pass the class name e.g. 'results__helper'?